cli/types/cypress.d.ts

```diff
@@ -1801,7 +1801,7 @@ declare namespace Cypress {
      *
      * @see https://on.cypress.io/reload
      * @example
-     *    cy.reload()
+     *    cy.reload({log: false, timeout: 30000})
      */
     reload(options?: Partial<Loggable & Timeoutable>): Chainable<AUTWindow>
     /**
```

Comment:
To make this the most complete, you can add another overload that shows that it takes no parameters, then change this overload so that the `options` parameter is not optional.  Then you can properly document both. This overload that takes an `options` should have an `@param` describing it.

Reply:
done, thanks for the suggestion!